backend/backend.go

```diff
@@ -34,7 +36,7 @@ func New(options Options) (BackendAPI, error) {
 	case s3Name:
 		return s3.New(options.Region, options.S3BucketName)
 	case ssmName:
-		return ssm.New(options.Region)
+		return ssm.New(options.Region, options.AWSConfig)
```

Comment:
shouldn't we add this also to S3 provider?

Reply:
yea, but I don't care about s3 provider :)